style.css

```diff
@@ -0,0 +1,636 @@
+  font-size: 25px;
+  margin: 5px 10px;
+  font-family: 'Inter', sans-serif;
+}
+
+.butt {
+  text-align: center;
+}
+
+.dis {
+  display: flex;
+  justify-content: center;
+}
+
+.about {
+  text-align: center;
+  font-size: 30px;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.about2 {
+  text-align: center;
+  padding: 15px 20px;
+  font-size: 23px;
+  background-image: url(./images/illustration\ 1-\ about\ me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: top right;
+  font-family: 'Inter', sans-serif;
+}
+
+.butt2 {
+  text-align: center;
+  background-color: #36b37e;
+  color: white;
+}
+
+.groupimg {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.btn2 {
+  text-align: center;
+}
+
+.project2 {
+  border: 1px solid green;
+  margin: 30px;
+  padding: 20px 1px;
+  height: 250px;
+  background-color: #e2e6ee;
+  font-family: 'Inter', sans-serif;
+}
+
+.language-icon {
+  display: flex;
+  justify-content: center;
+}
+
+.lang-text {
+  display: flex;
+  justify-content: center;
+  font-size: 24px;
+  font-family: 'Inter', sans-serif;
+}
+
+.lang-text-list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 8px;
+  padding-left: 0;
+}
+
+.li-lang {
+  border: 5px solid #fff;
+  background-color: #fff;
+  color: #36b37e;
+  font-size: 12px;
+  font-family: 'Inter', sans-serif;
+}
+
+.get {
+  text-align: center;
+  border: 2px solid #36b37f;
+  background-color: #36b37f;
+}
+
+.last {
+  font-family: 'Inter', sans-serif;
+  font-style: italic;
+  font-size: 20px;
+  height: 189px;
+  font-weight: 400;
+  display: flex;
+  justify-content: center;
+}
+
+.touch {
+  font-family: 'Inter', sans-serif;
+  font-style: italic;
+  font-size: 40px;
+  text-align: center;
+  align-items: center;
+  margin-bottom: 58px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.textareas {
+  align-items: center;
+  text-align: center;
+}
+
+.btn-resume {
+  width: 159px;
+}
+
+.resume-container {
+  background-image: url(./images/Group.png);
+  background-repeat: no-repeat;
+  background-position: bottom left;
+}
+
+.skills-container {
+  gap: 20px;
+  justify-content: center;
+}
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+}
+
+ul {
+  list-style: none;
+}
+
+input {
+  padding: 1px;
+  margin-bottom: 24px;
+  width: 300px;
+  height: 48px;
+}
+
+textarea {
+  padding: 0%;
+  margin-bottom: 24px;
+  width: 305px;
+  height: 114px;
+}
+
+.button {
+  height: 50px;
+  display: flex;
+  justify-content: center;
+}
+
+.bottom-line {
+  display: none;
+}
+
+.btn:active {
+  background-color: #008552;
+}
+
+.contact-form {
+  width: 55%;
+}
+
+.back-image {
+  height: 131px;
+  background-image: url(./images/illustration-contact-form.png);
+  background-repeat: no-repeat;
+  background-position: right;
+  width: 96vw;
+}
+
+.placeholdermargin {
+  padding: 0.1px 16px;
```

Comment:
- [ ] Kindly consider adding more top and bottom margin in order to fix it and make it similar to the figma design